feat: add manifest logo with dark mode display condition

SebConejo opened the pull request:
Here are the logos :
We need theme to be added to `https://eemeli.org/yaml/images/` repo.

![manifest-light](https://github.com/user-attachments/assets/f356546a-344d-4ac6-a6bb-d92b242e50e3)
![manifest-transparent](https://github.com/user-attachments/assets/e4df687f-9d8b-4c8c-98e3-d51e8fe758d1)

eemeli commented:
Hi! Could you file this PR in [eemeli/yaml-docs](https://github.com/eemeli/yaml-docs) instead? That's where the [image](https://github.com/eemeli/yaml-docs/blob/master/source/images/manifest.png) needs to go. Note also that GitHub's readme rendering is really limited, and e.g. selecting an image based on the light/dark mode is not possible. This means that you'll need to pick one image to use.

SebConejo commented:
@eemeli I have added the logo in svg format to the yaml-docs repo. So I updated the file here by replacing the png logo by a svg one, with a white background. So that we call only one logo for manifest.

eemeli commented:
Thank you! This one's much better in dark mode.
